### PR TITLE
feat: manage deps with renovate

### PR DESCRIPTION
**Pull request by theseion**
Experimental support for updating versions in bake file.

### CODE REVIEW

**Review by fzipi — COMMENTED** [review]
Looking good. Can you also include the common config using [this syntax](https://github.com/coreruleset/website/blob/main/renovate.json)?

The idea is to centralize renovate config in the https://github.com/coreruleset/renovate-config repo.

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      "depNameTemplate": "ModSecurity 3",
+      "customType": "regex",
+      "fileMatch": [
+        "^docker-bake.hcl$"
+      ],
+      "matchStrings": [
+        "^variable \"modsec3-version\" {\\n\\s+default = \"(?<currentValue> [^\"]+)\""
+      ],
+      "packageNameTemplate": "owasp-modsecurity/ModSecurity",
+      "datasourceTemplate": "github-releases"
```
Comment (fzipi) [inline renovate.json:58]: Is it smart enough to use the proper version? E.g. upgrading the corresponding major?

Comment (theseion) [reply renovate.json:58]: Yes, with the additional changes I've made.

**Comment by theseion** [comment]
I've figured out how to restrict ModSecurity 2 and how to also update the README automatically. I've tested it locally, but we'll only see what PRs are actually created once it runs for the first time.

**Review by fzipi — APPROVED** [review]
Excellent, let's try this out! 🚀

**Pull request merged**